session-1-jigsaw-intro/01_Greetings/es/README.md

```diff
@@ -0,0 +1,50 @@
+### Greetings example
```

Comment:
This is all English, are you sure you want it this way? Shouldn;t it be in Spanish?